e2e/node/manager_test.go

```diff
@@ -351,12 +352,6 @@ func (s *ManagerTestSuite) TestRaceConditions() {
 			progress <- struct{}{}
 		},
 		func(config *params.NodeConfig) {
-			log.Info("StopNode()")
```

Comment:
@b00ris Did you figure out why it's making the test unstable?

Reply:
@influx6 Whole test is unstable... I'm totaly agree with @JekaMas. 
The root case is: StopNode have time to close txPool before starting EventSystem eventLoop of  StartNode.

Reply:
Alright